flash-plugin.spec.in

```diff
@@ -2,81 +2,55 @@
 %global	        __strip /bin/true
 
 Name:           flash-plugin
-Version:        11.2.202.626
+Version:        27.0.0.130
 Release:        1%{?dist}
 Epoch:          1
 Summary:        Adobe Flash Player
 License:        Non-redistributable, no modification permitted
 URL:            http://get.adobe.com/flashplayer/
 ExclusiveArch:  %{ix86} x86_64
 
-Source0:        http://fpdownload.macromedia.com/get/flashplayer/pdc/%{version}/install_flash_player_11_linux.i386.tar.gz
-Source1:        http://fpdownload.macromedia.com/get/flashplayer/pdc/%{version}/install_flash_player_11_linux.x86_64.tar.gz
+%ifarch %{ix86}
+Source0:        https://fpdownload.adobe.com/get/flashplayer/pdc/%{version}/flash_player_ppapi_linux.i386.tar.gz
+%else
+Source1:        https://fpdownload.adobe.com/get/flashplayer/pdc/%{version}/flash_player_ppapi_linux.x86_64.tar.gz
+%endif
```

Comment:
I am downloading flash_player_ppapi_linux.x86_64.tar.gz , but we can both  package 
flash_player_Npapi_linux here and other with flash_player_Ppapi_linux, is what you mean ?